client/api_client.go

```diff
@@ -156,6 +147,9 @@ type ApiClientInterface interface {
 	PutEnvironmentDiscovery(projectId string, payload *EnvironmentDiscoveryPutPayload) (*EnvironmentDiscoveryPayload, error)
 	GetEnvironmentDiscovery(projectId string) (*EnvironmentDiscoveryPayload, error)
 	DeleteEnvironmentDiscovery(projectId string) error
+	TeamRoleAssignmentCreateOrUpdate(payload *TeamRoleAssignmentCreateOrUpdatePayload) (*TeamRoleAssignmentPayload, error)
```

Comment:
We call it "upsert" in our side but nvm, your call

Reply:
yeah it's just a name I guess.
I actually took it from the header in this URL:
https://docs.env0.com/reference/roles-upsert-team-role-assignment

Reply:
yeah it's just a name I guess.
I actually took it from the header in this URL:
https://docs.env0.com/reference/roles-upsert-team-role-assignment
I'm fine either way honestly... it's just a "replace" for me in the editor. Let me know...

Reply:
keep it like that